pennylane/_qubit_device.py

```diff
@@ -490,6 +490,18 @@ def statistics(self, observables, shot_range=None, bin_size=None):
                 # matrix.
                 results.append(self.access_state(wires=obs.wires))
 
+            elif obs.return_type is VnEntropy:
+                if len(observables) > 1:
+                    raise qml.QuantumFunctionError(
+                        "The Von Neumann entropy cannot be returned in combination"
+                        " with other return types"
+                    )
```

Comment:
The reasoning is that we required this for `qml.state` and as we are accessing the state I was thinking it would be logical to also add it here, but let me check that.

Reply:
I talked with @josh146 and we agree that we should support multiple return for `vn_entropy`, so I will remove this constraint!